torch/_weights_only_unpickler.py

```diff
@@ -103,6 +103,11 @@ def _get_allowed_globals():
         torch._utils._rebuild_sparse_csr_tensor,
     ]:
         rc[f"torch._utils.{f.__name__}"] = f
+
+    # Default rebuild function
+    # Handles Tensor Subclasses, Tensor's with attributes.
+    # NOTE: It calls into above rebuild functions for regular Tensor types.
+    rc["torch._tensor._rebuild_from_type_v2"] = torch._tensor._rebuild_from_type_v2
```

Comment:
What is that file/function?

Reply:
https://github.com/pytorch/pytorch/blob/20fc474410f49a8b9707297ce778425541a9a131/torch/_tensor.py#L57

Didn't understand the exact question/comment.

Reply:
Just asking what is this file doing? :p I've never seen it before.

Reply:
Ah. This was recently added to allow only unpickling of specific types (to partially fix unsafe load using pickle)

See https://github.com/pytorch/pytorch/pull/86812

Reply:
Ho ok, thanks!